docs/source/quickstart-aws.rst

```diff
@@ -44,61 +51,34 @@ Get ElasticBLAST
 
 
-The code examples below assume that ElasticBLAST was installed in the current
-working directory and that the :ref:`requirements <requirements>` have been met.
-
-
-Set up an output bucket (if one doesn't exist)
-----------------------------------------------
-
-.. code-block:: shell
-
-    aws s3 ls s3://elasticblast-${USER} || aws s3 mb s3://elasticblast-${USER}
+The instructions in this quickstart assume that you are working from the directory where you installed ElasticBLAST.
 
+Run the two ElasticBLAST commands listed below.  If ElasticBLAST is properly installed, the first one will report the version of ElasticBLAST installed and the second one will give you the help message.
 
+.. code-block:: bash
 
-Provide AWS credentials
------------------------
+    ./elastic-blast --version
+    ./elastic-blast --help
 
-If not already present, please please provide your AWS credentials. This can be
-accomplished by setting up environment variables or by saving those values in
-``~/.aws/config``. Please see examples below:
 
-.. code-block:: shell
-
-    # Environment variable
-    export AWS_ACCESS_KEY_ID=<YOUR_ACCESS_KEY_ID>
-    export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
+Set up an output bucket (if one doesn't exist)
+----------------------------------------------
 
 .. code-block:: shell
 
-    # Check whether an AWS configuration file already exists
-    [ -f ~/.aws/config ] || echo "AWS configuration file already exists!"
+    aws s3 ls s3://elasticblast-YOURNAME || aws s3 mb s3://elasticblast-YOURNAME
 
-    # If not, enter the following information in it
-    [ -d ~/.aws ] || mkdir ~/.aws
-    echo '[default]' > ~/.aws/config
-    echo 'aws_access_key_id = <YOUR_ACCESS_KEY_ID>' >> ~/.aws/config
-    echo 'aws_secret_access_key = <YOUR_SECRET_ACCESS_KEY>' >> ~/.aws/config
+Substitute your name for "YOURNAME"
 
 
 Configure ElasticBLAST
 ----------------------
 
-The minimal configuration requires: 
-
-#. :ref:`AWS region <elb_aws_region>` (``us-east-1`` recommended, see :ref:`AWS configuration <aws_conf>`),
-#. :ref:`query sequences <elb_queries>`,
-
-#. a :ref:`cloud storage bucket for results <elb_results>`. This value must start with ``s3://`` and _uniquely_ identifies your ElasticBLAST search. **Please keep track of this**.
-
-#. basic BLAST parameters (:ref:`program <elb_blast_program>` and :ref:`database <elb_db>`), and
-
-#. :ref:`elb_num_nodes` to start.
+You will use a configuration file to specify your input to ElasticBLAST.  Once you have written the configuration file, you'll just need to tell ElasticBLAST about it when invoked.
 
+Start by, copying the configuration file shown below.  Using an editor, write this text to a new file called "BDQA.ini".  Vi is pre-installed in the CloudShell.  Instructions for installing nano on the CloudShell can be found `here <https://docs.aws.amazon.com/cloudshell/latest/userguide/vm-specs.html#installing-software>`__.
```

Comment:
```suggestion
Start by, copying the configuration file shown below.  Using an editor, write this text to a new file called "BDQA.ini". `Vi <https://en.wikipedia.org/wiki/Vi>`_ is pre-installed in the CloudShell.  Instructions for installing nano on the CloudShell can be found `here <https://docs.aws.amazon.com/cloudshell/latest/userguide/vm-specs.html#installing-software>`__.
```

Reply:
I don't think we need a wikipedia link for vi.  If folks don't know what it it, they can google it.  For most users it becomes a link you click on and then realize you didn't need it.